SPECS/azurelinux-release/azurelinux-release.spec

```diff
@@ -118,6 +118,9 @@ install -Dm0644 %{SOURCE4} -t %{buildroot}%{_sysctldir}/
 %{_sysctldir}/*.conf
 
 %changelog
+* Fri Sep 27 2024 CBL-Mariner Servicing Account <rachelmenge@microsoft.com> - 3.0-20
```

Comment:
should this really be from "CBL-Mariner Servicing Account"?

Reply:
Nope. Copy paste error. Updating now